lib/6to5/transformation/templates/get.js

```diff
@@ -0,0 +1,23 @@
+(function get(object, property, receiver) {
+  var desc = Object.getOwnPropertyDescriptor(object, property);
+
+  if (desc === void 0) {
```

Comment:
I don't know, I just copied it from esnext as is. I might argue that generated code should use `void 0` to be more defensive, but if there's already a policy in place I would defer to that.